docs/pivy-importer.md

```diff
@@ -2,10 +2,27 @@
 
 The pivy-importer is a Java application that can be used to pull libraries from pypi into an Ivy format. It has
 several features that you may find useful in dealing with pypi projects.
+
+## Getting the Jar
+
+### Published Jar
+
+We publish the importer into bintray. You can find all the copies at https://bintray.com/linkedin/maven/pivy-importer.
+
+At the time of writing this doc, the current version is 0.3.16, so the jar you would want to download is
+https://dl.bintray.com/linkedin/maven/com/linkedin/pygradle/pivy-importer/0.3.16/:pivy-importer-0.3.16-all.jar .
+This artifact has all dependencies bundled together to make a nice CLI. 
+
+### Building it Locally
+
+To build the pivy-importer you'll need to run `./gradlew build` in the pivy-importer package. This will product several
+jars in `build/pivy-importer/libs/`. We product a normal jar that could be used when consuming in other projects and a 
+fat jar, with all the dependencies added into a single executable. This one is the one you would use on the commandline.
+It follows the naming convention `pivy-importer-<VERSION>-all.jar`.
  
 ## General Usage
 
-`java -jar pivy-importer.jar --repo /path/to/destination virtualenv:15.0.1 pip:7.1.2`
+`java -jar pivy-importer-VERSION-all.jar --repo /path/to/destination virtualenv:15.0.1 pip:7.1.2`
```

Comment:
will be great to have a full runnable example here. This command right now gives error about `Unable to find version 0.7 for alabaster` and `Unable to find version 0a for pytz`; that is, all the libs that require `--replace`.